src/oemof/solph/flows/_flow.py

```diff
@@ -86,7 +86,8 @@ class Flow(on.Edge):
         :class:`~oemof.solph.flows._simple_flow_block.SimpleFlowBlock`.
     fixed_costs : numeric (iterable or scalar), :math:`c_{fixed}`
         The fixed costs associated with a flow.
-        Note: These are only applicable for a multi-period model.
+        Note: These are only applicable for a multi-period model
+        and given on a yearly basis.
```

Comment:
That has been a discussion in a very old issue #556. I generally like the idea as well, but did not do so, because my intention was to add functionality, not to take major design choices for the existing framework.